modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,314 @@
+      },
+      xl: {
+        backgroundColor: 'Blue',
+      },
+    },
+    box: {
+      padding: 's',
+    },
+  },
+  width
+);
+
+return (
+  <Box ref={ref}>
+    <Flex {...containerResponsiveStyles.flex}>
+      <Box {...containerResponsiveStyles.box}>Hello World</Box>
+    </Flex>
+  </Box>
+);
+```
+
+### Viewport Based Responsive Styling
+
+The viewport is the user's visible area of a web page and it varies with screen size.
+
+We can use media queries to run a series of tests that will measure the screen size or resolution
+and apply CSS styles specifically to an element.
+
+An example of CSS media queries is below:
+
+```css
+@media screen and (min-width: 800px) {
+  .container {
+    margin: 1em 2em;
+  }
+}
+```
+
+In the example below, viewport-based styles are being applied by the size of the user's viewport.
+This is by using the breakpoint utilities (`up(), down(), between() and only()`). At the top of the
+code you will see `up() and down()` being used inside of the `StyledHeading` component. These
+utilities will be explained more in depth below the example. The size of the browser will have to be
+adjusted to see the changes take place on the UI Layout.
+
+<ExampleCodeBlock code={ResponsiveViewport} />
+
+### Applying Viewport-Based Responsive Styling
+
+We have three utilities to apply viewport-based responsive styling.
+
+`up() | down() | between() | only()`
+
+Each has a specific role in applying viewport-based responsive styles.
+
+`up()` - Returns a media query above the `min-width` for the range of a given breakpoint. Given a
+`start` breakpoint key `("zero", "s", "m", "l", "xl")`, this function returns a media query (string)
+using a `min-width`. See example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {up} = theme.canvas.breakpoints;
+const mediaQuery = up('l'); // Returns '@media (min-width: 1024px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.m,
+  },
+};
+```
+
+`down()` - Returns a media query below the `max-width` for the range of a given breakpoint. Given an
+`end` breakpoint key `("zero", "s", "m", "l", "xl")`, this function returns a media query (string)
+using a `max-width`.
+
+> **Note**: This function subtracts `0.5px` from the next breakpoint value to prevent collisions.
+> For example, `breakpoints.values.s`, has a `min-width` of `320px`, and the `max-width` is
+> `767.5px`). If the `xl` breakpoint is provided, this function returns a media query with only a
+> `min-width` of `0`, as seen in the second example below. See example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {up} = theme.canvas.breakpoints;
+const mediaQuery = up('l'); // Returns '@media (min-width: 1024px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.m,
+  },
+};
+```
```

Comment:
This looks like an unintended duplicate of the example for `up()` above. Can you double-check this and remove it if this is the case?

Also, my eyes are starting to glaze over from reading docs and this Breakpoints Utilities section is particularly monotonous (through no fault of yours, these utilities just all look the same) 😅 . Can you verify there aren't similar issues throughout this section?

Reply:
This was a duplicate and should have been showing the down example. 

The utilities do all look similar ha I agree.